extpsdk/requirements-sdk.in

```diff
@@ -1,5 +1,5 @@
 websockets>=12.0
 aiohttp>=3.9.5
-urllib3>=2.0.7
+urllib3>=2.2.2
```

Comment:
I don't think we need this part of the change...

Reply:
removed specification of urllib3 altogether.  So should be just in  requirements.txt now.